web/src/app/users/UserOnCallAssignmentList.tsx

```diff
@@ -85,11 +85,12 @@ export default function UserOnCallAssignmentList(props: {
   currentUser?: boolean
 }): JSX.Element {
   const userID = props.userID
-  const { data, loading, error } = useQuery(query, {
+  const [{ data, fetching, error }] = useQuery({
+    query,
     variables: { id: userID },
   })
 
-  if (!data && loading) {
+  if (!data && fetching) {
     return <Spinner />
   }
```

Comment:
Same as above, I think we can remove the use of `fetching`

For ref: when using Suspense loading, the `useQuery` function (but _not_ `useMutation`) will actually `throw` (a promise) rather then return `fetching = true`.